packages/webdoc-default-template/src/app/components/Explorer/ExplorerCategoryItem.js

```diff
@@ -1,23 +1,19 @@
 import ExplorerItem from "./ExplorerItem";
-import type {ExplorerTargetData} from "./ExplorerTargetData";
 import TreeItem from "@material-ui/lab/TreeItem";
 import cuid from "cuid";
 import {useExplorerStyles} from "./useExplorerStyles";
 
-export type ExplorerTargetGroupProps = {
-  title: string,
-  data: ExplorerTargetData[]
-}
 
-export default function ExplorerCategoryItem(props: ExplorerTargetGroupProps) {
+export default function ExplorerCategoryItem(props) {
```

Comment:
Why are you removing Flow types?

Reply:
For now they are crashing my IDE (I'm looking for fix on this), that's what i meant that after that we'll have to cleanup types.

Reply:
After merge I'll open issue for that.